libs/postgresql/Makefile

```diff
@@ -15,10 +15,9 @@ PKG_LICENSE:=PostgreSQL
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE_URL:=\
-	http://ftp9.us.postgresql.org/pub/mirrors/postgresql/source/v$(PKG_VERSION) \
-	http://ftp.be.postgresql.org/postgresql/source/v$(PKG_VERSION) \
+	https://ftp.postgresql.org/pub/source/v$(PKG_VERSION) \
```

Comment:
Change the list to the following as Postgresql seems to have dropped specific domain names for mirrors nowdays (drop ftp-archives).
https://ftp.postgresql.org/pub/source/v$(PKG_VERSION)
http://ftp.postgresql.org/pub/source/v$(PKG_VERSION)
ftp://ftp.postgresql.org/pub/source/v$(PKG_VERSION)